crates/cli/build.rs

```diff
@@ -13,7 +13,9 @@ fn main() -> Result<()> {
         let engine_path = Path::new(env!("CARGO_MANIFEST_DIR"))
             .join("../../target/wasm32-wasip1/release/core.wasm");
         println!("cargo:rerun-if-changed={}", engine_path.to_str().unwrap());
-        fs::copy(&engine_path, &destination)?;
+        wasm_opt::OptimizationOptions::new_opt_level_3() // Aggressively optimize for speed.
```

Comment:
Could we add a `TODO` comment here to state the effects of wasm-opt/reference types and to investigate a bit more why https://github.com/Shopify/ruvy/pull/92/files doesn't work? That PR is the ideal solution IMHO. However, I think the current approach is fine, given the the beta state of this project.

Reply:
Created #94 to investigate further